File: Module/PlayWays/src/main/java-room-grab/com/module/playways/grab/room/songmanager/model/RecommendTagModel.java
package com.module.playways.grab.room.songmanager.model;

import java.io.Serializable;

public class RecommendTagModel implements Serializable {

    /**
     * type : 123
     * name : 热门排行1
     */

    private int type;
    private String name;

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: Module/PlayWays/src/main/java-songmanage/com/module/playways/songmanager/model/RecommendTagModel.kt
package com.module.playways.songmanager.model

import java.io.Serializable

class RecommendTagModel : Serializable {

    // todo 测试先改一下，让服务器改回来
    /**
     * type : 123
     * name : 热门排行1
     */
    var type: Int = 0
    var name: String? = null

    // todo 排麦房临时这么用
    var tabDesc: String? = null
    set(value) {
        name = value
    }
    var tabType: Int = 0
    set(value) {
        type = value
    }
}
